### PR TITLE
Getting ready for our first release

### LINKED ISSUE #10: Cut a release
I've been developing this willy-nilly without a changelog or anything, but if other people want to use it, it would be best to include a changelog and release numbers so we can all track improvements and bug fixes. Also, adding a changelog is a great excuse to add Danger integration 😉.

### PR DESCRIPTION
**Pull request by ashfurrow**
This adds a changelog and Danger integration to make sure it gets updated.

Fixes #10.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Initially got a "not a pr, skipping" message, but after the build finished I did a `git commit --amend --no-edit ; git push -f` and it seemed to trigger Circle to admit that it was building a PR:

screen shot 2016-07-03 at 4 29 26 pm

**Comment by ashfurrow** [comment]
🍏 /cc @orta

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Warn about un-updated tests
+if has_app_changes && !has_test_changes
+  warn 'Tests were not updated'
+end
+
+if github.pr_body.length < 5
+  fail 'Please provide a summary in the Pull Request description'
+end
+
+# TODO: This could be a danger plugin
```
Comment (orta) [inline Dangerfile:28]: yep 👍

**Comment by orta** [comment]
yep - this all looks good to me

File: Dangerfile (hunk 1 above)
Comment (ashfurrow) [inline Dangerfile:28]: 💫 https://github.com/ashfurrow/danger-rubocop

Comment (orta) [inline Dangerfile:28]: I'll probably do some work on the danger-template, might be worth holding off for a day or two

Comment (ashfurrow) [inline Dangerfile:28]: Ah, just wrapping up tests 😉 At the rate it's going though, I'm sure I'll be making another one soon.

**Pull request merged**